features/step_definitions/money_manager_steps.rb

```diff
@@ -0,0 +1,22 @@
+Given(/^I am on the Money Manager tool$/) do
```

Comment:
Money Manager is the name of the tool rather than a clear description of the purpose for this feature. It's always helpful to match the feature name to the step definitions. 

I.e. `features/clear_data_on_signout.feature` should be expected to have it's step definitions defined within `features/step_definitions/clear_data_on_signout_steps.rb`